lookout/style/format/features.py

```diff
@@ -241,7 +245,7 @@ def _init_feature_names(self, feature_layout):
         self._feature2index = {name: i for i, name in enumerate(self.feature_names)}
 
     def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
-                         ) -> Optional[Tuple[numpy.ndarray, numpy.ndarray, List[VirtualNode]]]:
+                         ) -> Tuple[numpy.ndarray, numpy.ndarray, List[VirtualNode]]:
```

Comment:
But I see we can return `None` here: https://github.com/src-d/style-analyzer/pull/156/files#diff-f778fb4562b4ded91223ccd22dd95b08L279. So it is optional or 
we can fix return value to `return numpy.empty(shape=(0, 1)), numpy.empty(shape=(0, self.count_features(FeatureType.all))), []` because we do `a,b,c = fe.extract_features(...)` and it can failed.

Reply:
No no you're right it's an oversight during refactoring, thanks for catching it :+1: